nautobot/dcim/tests/test_views.py

```diff
@@ -381,6 +397,46 @@ def setUpTestData(cls):
             "comments": "New comments",
         }
 
+        # Create Devices
+        manufacturer = Manufacturer.objects.create(name="Manufacturer 1", slug="manufacturer-1")
+
+        device_types = (
+            DeviceType.objects.create(model="Device Type 1", slug="device-type-1", manufacturer=manufacturer),
+            DeviceType.objects.create(model="Device Type 2", slug="device-type-2", manufacturer=manufacturer),
+        )
+
+        device_roles = (
+            DeviceRole.objects.create(name="Device Role 1", slug="device-role-1"),
+            DeviceRole.objects.create(name="Device Role 2", slug="device-role-2"),
+        )
+
+        platforms = (
+            Platform.objects.create(name="Platform 1", slug="platform-1"),
+            Platform.objects.create(name="Platform 2", slug="platform-2"),
+        )
```

Comment:
Doesn't look like you're actually using the second object in each of these lists - might be simpler to just create a single instance each of DeviceType, DeviceRole, and Platform instead.